std_msgs/Header header
std_msgs/Bool front_left
std_msgs/Bool front_right
std_msgs/Bool rear_left
std_msgs/Bool rear_right
std_msgs/Bool side_front_left
std_msgs/Bool side_rear_left
std_msgs/Bool side_front_right
std_msgs/Bool side_rear_right


